### PR TITLE
PHP: Surface a static method to fetch the default Channel

### PR DESCRIPTION
**Pull request by chingor13**
Prior to having access to interceptors, a developer didn't care about the default Channel created.

If you are using interceptors and you want to use and interceptor with the default channel you would have to replicate the code in the BaseStub constructor.

This PR exposes a static method `BaseStub::getDefaultChannel(array $opts)` that can be used to fetch the default channel so that a developer can wrap the channel with an interceptor.

@ZhouyihaiDing

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @chingor13 isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by chingor13** [comment]
I signed it!

File: src/php/lib/Grpc/BaseStub.php [hunk 1]
```diff
@@ -69,6 +69,18 @@ public function __construct($hostname, $opts, $channel = null)
...
+    }
+
+    /**
+     * Creates and returns the default Channel
+     *
+     * @param array $opts Channel constructor options
+     *
+     * @return Channel The channel
+     */
+    public static function getDefaultChannel(array $opts)
```
Comment (ZhouyihaiDing) [inline src/php/lib/Grpc/BaseStub.php]: 'getDefaultChannel($hostname, array $opts)'?

Comment (chingor13) [inline src/php/lib/Grpc/BaseStub.php]: Could also make this `public static function getDefaultChannel(ChannelCredentials $credentials, array $opts = [])` as the credentials are required.

Comment (chingor13) [reply src/php/lib/Grpc/BaseStub.php]: ah, yes, forgot `$hostname`

Comment (ZhouyihaiDing) [reply src/php/lib/Grpc/BaseStub.php]: I think it's okay to leave it inside the `$opts`. C extensions will read `$opts['credentials']`.

**Comment by ZhouyihaiDing** [comment]
known issue: #15425

**Pull request merged**